portfolio/src/main/java/com/google/sps/data/Nickname.java

```diff
@@ -11,7 +11,8 @@
     /**
```

Comment:
When a class only exists to provide static methods/constants, and there's no point to anyone creating instances of it, it's best-practice to declare a private empty constructor, like `private Nickname() {}`. This makes it so that any other code using that class can't accidentially create an instance of it it, since then doing `new Nickname()` from anywhere else would be an error. Note that a class which doesn't have an explicit constructor implicitly has a default-visibility constructor as though you wrote `Nickname() {}`.

Reply:
I forgot to address this in the last revision, but thanks for providing an explanation.